docs/kubernetes-cluster-management/resource-usage.md

```diff
@@ -0,0 +1,19 @@
+# Overview
+Predictable demands are important to help Kubernetes find the right place for a pod within a cluster.  When it comes to resources such as a CPU and memory, understanding the resource needs of a pod will help Kubernetes allocate the pod to the most appropriate node and generally improve the stability of the cluster.
+
+# FFC resource usage standards
+- all pods should declare both a `reqeuest` and `limit` value for CPU and memory
```

Comment:
typo, `reqeuest` should be `request`. Presumably a pod should only be 'burstable' on network bandwidth if we always declare CPU and memory limits?

Reply:
Fixed typo.  A pod would be still be considered burstable if we specified request and limit, but limit was higher than request.  That means the pods usage isn't fixed and therefore introduces more risk of instability to the cluster.